template_src/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "my_app",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "none specified",
+  "license": "none specified"
```

Comment:
which ones?
name and version get filled in if those fields are specified the `cordova create` command
I could get rid of all the other variables since I don't know what the default values should be